photon-client/src/components/settings/DeviceControlCard.vue

```diff
@@ -201,6 +201,39 @@ const handleSettingsImport = () => {
   importType.value = -1;
   importFile.value = null;
 };
+
+const showFactoryReset = ref(false);
+const expected = "I solumnly swear that I am up to no good";
```

Comment:
It's "solemnly"

Reply:
lol. I'm gunna change this to something a little less quirky, maybe like "Yes I backed up my settings"? Idk if "swear" would get us in trouble with some people lol